### PR TITLE
WIP: Add GstLibav using the default system ffmpeg

### PR DESCRIPTION
**Pull request by thiblahute**
Same idea as https://gitlab.gnome.org/GNOME/gnome-sdk-images/merge_requests/3

Marking as WIP as it has not been directly testing, is there a way to get that going through some CI?

### CODE REVIEW

File: org.freedesktop.Sdk.json.in [hunk 1]
```diff
@@ -2208,6 +2208,31 @@
                  }
             ]
         },
+        {
+            "name": "gstreamer-libav",
```
Comment (TingPing) [inline org.freedesktop.Sdk.json.in:2212]: ~~Why not build with `meson`~~

EDIT: Nevermind this is a very old gstreamer here, forgot.

File: org.freedesktop.Sdk.json.in [hunk 2]
```diff
@@ -2208,6 +2208,31 @@
...
+            "build-options" : {
+                "arch" : {
+                    "i386" : {
+                        "config-opts" : [
+                            "--build=i586-unknown-linux-gnu"
```
Comment (TingPing) [inline org.freedesktop.Sdk.json.in:2217]: Dozens of packages have been building without these, don't think they are required?

Comment (thiblahute) [reply org.freedesktop.Sdk.json.in:2217]: Not sure what those are for, but follow what seems to be done for all other GStreamer plugins actually. Shoulds we just try without?

Comment (TingPing) [reply org.freedesktop.Sdk.json.in:2217]: It is probably harmless.

**Pull request merged**